webrender/src/internal_types.rs

```diff
@@ -146,18 +146,18 @@ impl GLContextWrapper {
 
                 let limits = ctx.borrow_limits().clone();
 
-                (real_size, texture_id, limits)
+                (DeviceIntSize::from_untyped(&real_size), texture_id, limits)
             }
         }
     }
 
-    pub fn resize(&mut self, size: &Size2D<i32>) -> Result<(), &'static str> {
+    pub fn resize(&mut self, size: &DeviceIntSize) -> Result<(), &'static str> {
```

Comment:
it rubs me the wrong way to see `Int` there, but I can see that it was this way before you